### PR TITLE
fix typos on email

### PR DESCRIPTION
**Pull request by alejandroGonGon**

### CODE REVIEW

File: resources/views/emails/donationRejected.blade.php [hunk 1]
```diff
@@ -95,7 +95,7 @@
...
-</td></tr><tr><td align="left" class="x " style="font-size:0;padding:0;padding-bottom:0;word-break:break-word;"><div style="font-family:Roboto,Arial,sans-serif;font-size:16px;line-height:24px;text-align:left;color:#000000;"><p style="Margin:0;text-align:left;"><span style="mso-line-height-rule:exactly;font-size:16px;font-family:Roboto,Arial,sans-serif;font-weight:400;color:#777777;line-height:24px;">{{$donationData['reasonRejected']}} </span></p></div>
+</td></tr><tr><td align="left" class="x " style="font-size:0;padding:0;padding-bottom:0;word-break:break-word;"><div style="font-family:Roboto,Arial,sans-serif;font-size:16px;line-height:24px;text-align:left;color:#000000;"><p style="Margin:0;text-align:left;"><span style="mso-line-height-rule:exactly;font-size:16px;font-family:Roboto,Arial,sans-serif;font-weight:400;color:#777777;line-height:24px;">Tu donación ha sido rechazada por el equipo debido a que no cumple con los estándares esperados, por cualquier inconveniente comunicarse a este mail : <strong>hoydonamos@gmail.com<strong></span></p></div>
```
Comment (nicocadq) [inline resources/views/emails/donationRejected.blade.php:98]: Add email into an env

File: resources/views/emails/donationPublished.blade.php [hunk 2]
```diff
@@ -117,7 +117,7 @@
...
+<table border="0" cellpadding="0" cellspacing="0" style="border-collapse:separate;width:145px;line-height:100%;"><tbody><tr><td align="center" bgcolor="#73b8b2" style="border:none;border-radius:8px 8px 8px 8px;cursor:auto;mso-padding-alt:12px 0px 12px 0px;background:#73b8b2;" valign="middle"> <a href="http://ec2-3-145-124-177.us-east-2.compute.amazonaws.com/" style="display:inline-block;width:145px;background:#73b8b2;color:#ffffff;font-family:Roboto,Arial,sans-serif;font-size:13px;font-weight:normal;line-height:100%;margin:0;text-decoration:none;text-transform:none;padding:12px 0px 12px 0px;mso-padding-alt:0;border-radius:8px 8px 8px 8px;" target="_blank"> <span style="mso-line-height-rule:exactly;font-size:14px;font-family:Roboto,Arial,sans-serif;font-weight:700;color:#ffffff;line-height:16px;">VER DONACIÓN</span></a>
```
Comment (nicocadq) [inline resources/views/emails/donationPublished.blade.php:120]: Wrong URL at the `a` tag

**Review by nicocadq — APPROVED** [review]

File: resources/views/emails/donationRejected.blade.php (hunk 1 above)
Comment (alejandroGonGon) [reply resources/views/emails/donationRejected.blade.php]: done

File: resources/views/emails/donationPublished.blade.php (hunk 2 above)
Comment (alejandroGonGon) [reply resources/views/emails/donationPublished.blade.php]: done

**Pull request merged**